### PR TITLE
feat(sdk-coin-coredao): add coredao sdk skeleton

### PR DESCRIPTION
**Pull request by adarsh312**
Ticket: WIN-3696

### CODE REVIEW

File: modules/sdk-coin-coredao/src/lib/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import * as Utils from './utils';
+
+export { TransactionBuilder } from './transactionBuilder';
+export { TransferBuilder } from './transferBuilder';
```
[the remark this answers is not in the document]
Comment (adarsh312) [reply modules/sdk-coin-coredao/src/lib/index.ts:4]: do we need them ? as i am using them anywhere

[the remark this answers is not in the document]
Comment (gianchandania) [reply modules/sdk-coin-coredao/src/lib/index.ts:4]: Mainly it is being used in the tests, so you can add it later when you add tests

**Review by gianchandania — DISMISSED** [review]

**Review by gianchandania — APPROVED** [review]

File: modules/sdk-coin-coredao/src/coredao.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  /** @inheritDoc */
+  supportsTss(): boolean {
+    return true;
+  }
+
+  /** @inheritDoc */
+  getMPCAlgorithm(): MPCAlgorithm {
+    return 'ecdsa';
```
Comment (margueriteblair) [inline modules/sdk-coin-coredao/src/coredao.ts:26]: nit: do we not have some type of const that can be used instead of the hardcoded string?

**Review by margueriteblair — APPROVED** [review]

**Review by sumanmukherjee03 — APPROVED** [review]
approving the codeowners change

**Pull request merged**